apps/dashboard/app/models/batch_connect/app.rb

```diff
@@ -161,11 +162,21 @@ def clusters
       OodAppkit.clusters.select { |cluster| cluster_allowed(cluster) }
     end
 
+    def preset?
+      valid? && attributes.all?(&:fixed?)
+    end
+
     # Whether this is a valid app the user can use
     # @return [Boolean] whether valid app
     def valid?
       if form_config.empty?
         false
+      elsif !form_config.fetch(:form, []).is_a?(Array)
+        @validation_reason = I18n.t('dashboard.batch_connect_invalid_form_array')
+        false
+      elsif !form_config.fetch(:attributes, {}).is_a?(Hash)
+        @validation_reason = I18n.t('dashboard.batch_connect_invalid_form_attributes')
+        false
```

Comment:
I think this is a part of #1812 which I'm not sure if we want to back port.

Reply:
> I think this is a part of #1812 which I'm not sure if we want to back port.


This has to be back-ported otherwise the code fails.  Other than the base Preset App functionality, I only back-ported other functionality if there was error due to it missing. Which there were a couple.  This was one of them.

Reply:
I'm anxious about porting more than we absolutely need. Every addition carries risk and right now tests are failing for ... reasons that are unclear.

I can try to look at what specifically was failing past the first commit.